src/portfolio/APIClient.ts

```diff
@@ -486,18 +503,21 @@ function convertPortfolioDetails(response: DetailsResponse): PortfolioDetails {
  * @param url - An URL relative to {@link baseURL}
  * @param body - The request body
  * @param headers - Additional request headers
+ * @param isAnalytics - if set to true the endpoint is changed to the analytics endpoint
```

Comment:
Wouldn't it make sense to use a string parameter that can be set to any endpoint to increase flexibility of the `request` method?